src/app/rosetta/start.sh

```diff
@@ -1,25 +1,47 @@
 #!/bin/bash
 
+set -eou pipefail
+
 function cleanup
 {
-  kill $(jobs -p)
-  /etc/init.d/postgresql stop
+  echo "Cleaning up backgrounded processes"
+  kill $(ps aux | egrep '_build.*archive\.exe' | grep -v grep | awk '{ print $2 }')
```

Comment:
Couldn't this match a variety of non-coda binaries run through dune too? Might be safer to pass the full path.